modules/modules.nix

```diff
@@ -113,6 +114,7 @@ let
     (loadModule ./services/nextcloud-client.nix { })
     (loadModule ./services/owncloud-client.nix { })
     (loadModule ./services/parcellite.nix { })
+    (loadModule ./services/password-store-sync.nix { condition = hostPlatform.isLinux; })
```

Comment:
I'm not sure whether the service module should have the same name as the program module. I suffixed it with `-sync` so that it's more explicit about what it does.

Reply:
I think the `-sync` suffix is good.